### PR TITLE
[Manta]Use type aliases to fix the issue about front-end loads the backend

### PR DESCRIPTION
**Pull request by Dengjianping**
Fix the error from front-end while load backend.
```
FATAL: Unable to initialize the API: [u8;PRIVATE_TRANSFER_PAYLOAD_SIZE]: Only support for [Type; <length>], where length <= 256
```

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -337,7 +341,7 @@ decl_module! {
 		/// Neither the values nor the identities is leaked during this process.
 		#[weight = T::WeightInfo::private_transfer()]
 		fn private_transfer(origin,
-			payload: [u8; PRIVATE_TRANSFER_PAYLOAD_SIZE],
+			payload: PrivatePayload,
```
Comment (zhenfeizhang) [inline src/lib.rs]: Can you please rename this to `PrivateTransferPayload`?

Comment (Dengjianping) [reply src/lib.rs]: Just renamed it.

**Review by stechu — APPROVED** [review]
LGTM. Can you also create a PR in `Manta` to update the `types.json`?

**Comment by Dengjianping** [comment]
Sure.

**Pull request merged**